orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/backup_db.sh

```diff
@@ -27,26 +27,71 @@ doLog() {
   echo "$(date "+%Y-%m-%dT%H:%M:%SZ") $type_of_msg ""$msg" >>$LOGFILE
 }
 
-if [[ $# -lt 6 || $# -gt 10 || "$1" == "None" ]]; then
-  doLog "Invalid inputs provided"
-  doLog "Script accepts at least 6 and at most 7 inputs:"
-  doLog "  1. Object Storage Service url to place backups."
-  doLog "  2. PostgreSQL host name."
-  doLog "  3. PostgreSQL port."
-  doLog "  4. PostgreSQL user name."
-  doLog "  5. Ranger admin group."
-  doLog "  6. Whether or not to close connections for the database while it is being backed up."
-  doLog "  7-10. (optional) Names of the databases to backup. If not given, will backup ranger and hive databases."
-  exit 1
-fi
+log_invalid_input_reason() {
+  doLog "This might be caused by the command is not running on the master Getaway node or there has never had a backup/restore from the CDP CLI run."
```

Comment:
Grammar fix: `"This might be caused by the command not being run on the Primary Gateway node or due to never having run a backup/restore via the CDP CLI before."`

Reply:
Updated this, thank you so much!